Configuration/FlexForms/flexforms_pi1.xml

```diff
@@ -350,6 +350,7 @@
 							<config>
 								<type>select</type>
 								<foreign_table>tx_seminars_event_types</foreign_table>
+				                <foreign_table_where>AND tx_seminars_event_types.pid LIKE ###PAGE_TSCONFIG_STR### ORDER BY title</foreign_table_where>
```

Comment:
If this is only a single UID, this shouldn't use a LIKE query, but instead a direct comparison. (Also, LIKE is relatively slow AFAIK.)